### PR TITLE
docs: add docs about creating data stores using the CLI

### PR DESCRIPTION
**Pull request by mathnogueira**
This PR adds documentation on how you can create data stores using the CLI.

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/docs/cli/creating-data-stores.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Defining Data Stores as Text Files
+You might have multiple tracetest instances that need to be connected to the same data stores. An easy way of sharing the configuration is by using a configuration file that can be applied to your tracetest instance.
```
Comment (jfermi) [inline docs/docs/cli/creating-data-stores.md:2]:
```suggestion
You might have multiple Tracetest instances that need to be connected to the same data stores. An easy way of sharing the configuration is by using a configuration file that can be applied to your Tracetest instance.
```

File: docs/docs/cli/creating-data-stores.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+spec:
+  name: SignalFX
+  type: signalFx
+  isDefault: true
+  signalFx:
+    realm: us1
+    token: mytoken
+```
+
+## Apply configuration
```
Comment (jfermi) [inline docs/docs/cli/creating-data-stores.md:55]:
```suggestion
## Apply Configuration
```

File: docs/docs/cli/creating-data-stores.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+To apply the configuration, you need a [configured CLI](./configuring-your-cli.md) pointed to the instance you want to apply the data store. Then you just have to enter:
+
+```
+tracetest data-store apply -f my/data-store/file/location.yaml
+```
+
+## Additional information
```
Comment (jfermi) [inline docs/docs/cli/creating-data-stores.md:63]:
```suggestion
## Additional Information
```

**Review by jfermi — COMMENTED** [review]
A few editing suggestions.

File: docs/docs/cli/creating-data-stores.md (hunk 3 above)
Comment (kdhamric) [reply docs/docs/cli/creating-data-stores.md:74]: Do we have a way to export the data store configuration yet? If so, we should document it as the best way to build a configuration file to be used elsewhere to import a configuration.

File: docs/docs/cli/creating-data-stores.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+### Using the OpenTelemetry Collector
+```yaml
+type: DataStore
+spec:
+  name: Opentelemetry Collector pipeline
+  type: otlp
+  isDefault: true
+```
+
+> Consider reading about [how to use the OTEL collector](../configuration/connecting-to-data-stores/opentelemetry-collector.md) to send traces to your tracetest instance.
```
Comment (danielbdias) [inline docs/docs/cli/creating-data-stores.md:64]:
```suggestion
> Consider reading about [how to use the OTEL collector](../configuration/connecting-to-data-stores/opentelemetry-collector.md) to send traces to your Tracetest instance.
```

File: docs/docs/cli/creating-data-stores.md (hunk 3 above)
Comment (mathnogueira) [reply docs/docs/cli/creating-data-stores.md]: No, we don't have it yet.

**Review by kdhamric — APPROVED** [review]

**Pull request merged**